input-output-hk/daedalus
# [DDW-753] Licensing update

This PR introduce license update to "Apache 2.0 license".

## Screenshots

![About -EN](https://user-images.githubusercontent.com/18653950/61278176-f9984000-a7b3-11e9-92ff-ed66fc900791.png)

![About - JA](https://user-images.githubusercontent.com/18653950/61278180-fbfa9a00-a7b3-11e9-96c5-af919bc938bd.png)

---

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1563265682380400)

- [ ] Test that Apache 2.0 license is applied to repository and match requirements from https://docs.google.com/document/d/1V93Qk_rFGMTRdZWcJid4jWZ4iRwNp8bZ_2J2NRydEWo/edit.
- check LICENSE https://github.com/input-output-hk/daedalus/blob/chore/ddw-753-licensing-update/LICENSE 
- check LICENSE NOTICE https://github.com/input-output-hk/daedalus/blob/chore/ddw-753-licensing-update/LICENSE%20NOTICE

- [ ] Test "About" screen - Check if English "copy" matches Japanese "copy" after changing "Daedalus" and "Cardano" words to original / untranslated  words.

---

## Review Checklist

### Basics


### Code Quality

### Testing

### After Review
